frontend/models/product.ts

```diff
@@ -19,8 +20,20 @@ export async function findProduct(shop: ShopCredentials, handle: string) {
       return null;
    }
    const variants = response.product.variants.nodes.map((variant) => {
+      const isDiscounted =
+         variant.compareAtPriceV2 != null &&
+         variant.compareAtPriceV2.amount > variant.priceV2.amount;
```

Comment:
I think these amounts are serialized as strings (e.g. "29.99") so we probably need to cast them as numbers before treating them as such.

Reply:
Good catch!